### DISCUSSION TITLE
Where to find uwebsocket (js) implementation examples?

### DISCUSSION THREAD

**Discussion by SOHAMXBR** [uNetworking/uWebSockets.js#1093] [General]
I have just integrated uwebsocket in my js project. But no way to go ahead. How to create http server?websocket server?recieve message? Send message? The documentation only have C++ implementation examples. I want to read documentation where from i can know how to implement these. But strange there are not a single js example stated anywhere in the documentation or other websites. How am I supposed to start my journey from here on?

**Comment by e3dio** [comment]
https://github.com/uNetworking/uWebSockets.js/tree/master/examples

**Reply by SOHAMXBR** [reply]
Thank u so much..i was searching in the c++ version. my bad
